src/Applications/GEOSdas_App/ldas_add2rc

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+MYNAME="ldas_add2rc"
+if [[ $# -ne 2 ]]; then 
+   echo " "
+   echo " NAME "
+   echo " "
+   echo "  $MYNAME  - reconcile GEOSldas and AGCM configurations in weakly-coupled LADAS "
+   echo " "
+   echo " SYNOPSIS  "
+   echo " "
+   echo "  $MYNAME  FVHOME  LDHOME"
+   echo " "
+   echo " where"
+   echo "   FVHOME  -  location of  ADAS experiment "
+   echo "   LDHOME  -  location of  LDAS experiment "
+   echo " "
+   echo " DESCRIPTION"
+   echo " "
+   echo "    This procedure inserts select GEOSldas resource parameters into AGCM.rc and " 
+   echo "    verifies consistency between LDAS.rc and AGCM.rc for other resource parameters "
+   echo " "
+   echo "   $MYNAME should be run after fvsetup and ldas_setup  "
+   exit 0
+fi
+
+FVHOME="$1" 
+LDHOME="$2" 
+
+# ----------------------------------------------------------------
+# extract the following parameters from LDAS.rc and add to AGCM.rc
+
+cd $FVHOME/run
+cp AGCM.rc.tmpl  AGCM.rc.tmpl.original 
+
+list="LANDASSIM_DT LANDASSIM_T0 "
+  for var in $list
+do
+    grep $var $LDHOME/run/LDAS.rc >> add.txt
+done
+
+sed -i '/LDAS_INCR/r add.txt' AGCM.rc.tmpl
+
+echo " The following land parameters are added to AGCM.rc: " 
+for var in $list
+do
+echo "$(grep $var add.txt)"
+done
+
+# add same resource parameters into AGCM.rc for atm ens
```

Comment:
I moved up the block that adds LANDASSIM_DT and LANDASSIM_T0 into AGCM.rc so it's together with the same action for the central AGCM.rc.  Not sure if the logic is still intact.
Also, what happens if the ADAS is run in 3D-Var configuration?  In this case, the ./atmens/ directory should not exist.  This script might need to know if the ADAS is in 3D-Var or in Hybrid 4D-EnVar and do its thing accordingly.

Reply:
the original sequence ( atmens at the last step) would exit however not gracefully when in case of 3D-var,  while for the central all checking/adding are already carried out.   I will add "3dvar" to the argument input, so when that is present the script should skip atmens part.

Reply:
I don't think we should have non-graceful exits, so we need to have an "if" statement for presence/absence of the atm ensemble.  
It makes sense to run through all parameter values and tabulate the ones that are inconsistent before exiting.
But what still seems to be missing is a "return code" and controlled exit.  This script should return something to the main script that tells the main script to exit with a message.  If the list of inconsistent parameters is written by this script, then the main script can simply exit with a brief error message, something like "ldas_add2rc: inconsistent ADAS and LDAS config"

Reply:
yes i will add 3dvar / hybrid  case choice  and the corresponding "if"  block.